src/algebra/hom/group.lean

```diff
@@ -1041,6 +1041,12 @@ add_decl_doc add_monoid_hom.has_zero
 @[simp, to_additive] lemma monoid_hom.one_apply [mul_one_class M] [mul_one_class N]
   (x : M) : (1 : M →* N) x = 1 := rfl
 
+@[simp, to_additive] lemma one_hom.coe_monoid_hom_one [monoid M] [monoid N] :
```

Comment:
```suggestion
@[simp, to_additive] lemma monoid_hom.to_one_hom_one [monoid M] [monoid N] :
```
but if we add this lemma, we should do the same for:
* `monoid_hom.to_mul_hom_mul`
* `linear_map.to_add_monoid_hom_zero`
* `linear_map.to_add_monoid_hom_add`